src/main/java/ru/javawebinar/topjava/util/UserMealsUtil.java

```diff
@@ -23,5 +23,44 @@ public static void main(String[] args) {
                 new UserMeal(LocalDateTime.of(2015, Month.MAY, 31, 13, 0), "Обед", 500),
                 new UserMeal(LocalDateTime.of(2015, Month.MAY, 31, 20, 0), "Ужин", 510)
         );
+
+        getFilteredWithExceeded(mealList, LocalTime.of(7, 0), LocalTime.of(12, 0), 2000);
+//        .toLocalDate();
```

Comment:
toLocalDate and toLokalTime releted to the next part of HW so it should't be here